app/views/dashboard/form/contributors/_authorship_fields.html.erb

```diff
@@ -5,18 +5,26 @@
 <div class="nested-fields contributor-wrapper js-contributor-wrapper">
   <div class="row">
     <div class="col-2">
+      <%= form.hidden_field :position, class: 'js-position-index' %>
       <span class="badge badge-dark badge--outline">
         <%= t('dashboard.form.contributors.edit.badge') %>
         <span class="badge--index"><%= index + 1 %></span>
       </span>
     </div>
 
+    <%= form.hidden_field :actor_id %>
+    <%= form.fields_for :actor do |actor_fields| %>
+      <%= actor_fields.hidden_field :email %>
+      <%= actor_fields.hidden_field :given_name %>
+      <%= actor_fields.hidden_field :surname %>
+      <%= actor_fields.hidden_field :psu_id %>
+      <%= actor_fields.hidden_field :orcid %>
```

Comment:
I can't figure out why we're doing these hidden fields, but I assume this is how we provide this information back to the controller when a new actor is added from the js/autocomplete?

Reply:
Yes that's correct. We don't ever update these fields, which makes me wonder if we should somehow _not_ render them in the form if the actor already exists?